src/App.jsx

```diff
@@ -18,7 +18,7 @@ function App() {
       </div>
       <h1>LnL Video App</h1>
       <div className='card'>
-        <video width={1920} height={1080} controls>
+        <video width={1280} height={720} controls>
```

Comment:
I can see that this looks correct but the video is slightly off center, please check

Reply:
Nice catch: I've updated the player position